crates/stark_hash/src/hash.rs

```diff
@@ -62,7 +62,11 @@ impl std::fmt::Display for OverflowError {
 }
 
 impl StarkHash {
-    pub const ZERO: StarkHash = StarkHash([0u8; 32]);
+    const ZERO: StarkHash = StarkHash([0u8; 32]);
+
+    pub const fn zero() -> Self {
+        StarkHash::ZERO
+    }
```

Comment:
Is there a benefit to hiding the const and preferring a fn for it? I'd be happy leaving the const pub.

Reply:
keeping `ZERO` as `pub`.